float64 gripper_left
float64 gripper_right